static/app/components/onboarding/productSelection.tsx

```diff
@@ -40,6 +41,7 @@ function getDisabledProducts(organization: Organization): DisabledProducts {
   const hasSessionReplay = organization.features.includes('session-replay');
   const hasPerformance = organization.features.includes('performance-view');
   const hasProfiling = organization.features.includes('profiling-view');
+  const isSelfHostedErrorsOnly = ConfigStore.get('isSelfHostedErrorsOnly');
 
   const reason = t('This feature is not enabled on your Sentry installation.');
```

Comment:
Can we not just remove the buttons entirely? I feel like by the time the user sees this, they will already be too far down the errors-only path (ie, they've selected it during install.sh) to do much about this now.

Reply:
We can remove the buttons entirely, and I considered doing that. The reason I kept the buttons is that as time goes on and we add more features, it would indicate to users that a certain feature that is new is disabled. Maybe then they would make the tradeoff of going to full self-hosted Sentry or SaaS to include this feature.

Reply:
done